pkg/shell/base_index.js

```diff
@@ -613,6 +623,50 @@
         return false;
     };
 
+    function setDarkMode(documentElement, dark_mode) {
+        if (dark_mode) {
+            documentElement.classList.add('pf-theme-dark');
+        } else {
+            documentElement.classList.remove('pf-theme-dark');
+        }
+    }
+
+    function updateFrames(dark_mode) {
+        setDarkMode(document.documentElement, dark_mode);
+        Object.values(self.frames.iframes).forEach(machine => {
+            Object.values(machine).forEach(frame => {
+                setDarkMode(frame.contentDocument.documentElement, dark_mode);
+            });
+        });
+    }
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17914-20221118-114012-da84543d-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)